libraries-bom/pom.xml

```diff
@@ -38,7 +38,7 @@
   <licenses>
     <license>
       <name>The Apache License, Version 2.0</name>
-      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <url>https://raw.githubusercontent.com/googleapis/google-cloud-java/main/LICENSE</url>
```

Comment:
Wasn't sure if there was significance of libraries-bom pointing directly to apache url vs google-cloud-java's LICENSE file

Reply:
Took a look at the official Maven docs at https://maven.apache.org/pom.html#licenses -- It doesn't seem to suggest anything regarding where the LICENSE points. 

I think both would work since they just point to contents of the LICENSE file and they're the same.